src/test/java/com/conveyal/datatools/manager/jobs/ArbitraryTransformJobTest.java

```diff
@@ -282,6 +314,10 @@ private static String generateStopsWithCustomFields() {
             + "\n1234567,customValue3,customValue4";
     }
 
+    private static String generateStopRow() {
+        return "\nnew,new,appended stop,,37.06668,-122.07781,,,0,123,,";
```

Comment:
Will the first row of data always start with a new line character? Knowing that the data is being appended there should probably be a check for this and if it is missing it should be added. I would like to see more than one row appended, perhaps 3 or 4 just to be sure of the new line counter.